README.md

```diff
@@ -8,7 +8,32 @@ Make without limit! Device Simulator Express, a Microsoft Garage project, allows
 result when you plug in your actual microcontroller. Curious about the output of the device, the serial
 monitor allows you to observe the device output.
 
-## Devices we support:
+## Table of Contents
+  - [Devices we support](#devices-we-support)

```

Comment:
Do we know how the table of contents will work in the marketplace?

Reply:
It works! The links are markdown links and we have some in the marketplace already for "devices we support". Clicking on a device takes you to the device part in the marketplace :)